Rocket.Chat.iOS/ViewControllers/ViewController.swift

```diff
@@ -323,14 +345,15 @@ class ViewController: UIViewController, UITableViewDataSource, UITableViewDelega
     }
     
     
+    
     //Function to close the keyboard when send button is pressed
     @IBAction func sendMsg(sender: AnyObject) {
         
         //If there is text
         if composeMsg.text != "" {
             //create current message
+//            let currentMsg:Message = Message(id: "", text: composeMsg.text, tstamp: NSDate(timeInterval: randomTime(), sinceDate: NSDate()), user: currentUser!)
```

Comment:
This is just to create dummy data. Offcourse this will be removed when we actually "talk" with the server